### PR TITLE
Use string interpolation

### PR DESCRIPTION
**Pull request by Youssef1313**
- Used string interpolation
- Used var instead of actual type when it's clear from the right side
- Modernizing sample

### CODE REVIEW

**Comment by Youssef1313** [comment]
@rpetrusha , I updated the samples you mentioned.
I also noticed that I get some different results when I run the code, can you check those commits ? (I still didn't check the parallel C# for those, I'll wait your review first)

File: snippets/visualbasic/VS_Snippets_CLR_Classic/classic Hashtable.Clear Example/VB/source.vb [hunk 1]
```diff
@@ -25,7 +24,7 @@ Public Class SamplesHashtable
 
         ' Displays the count and values of the Hashtable.
         Console.WriteLine("After Clear,")
-        Console.WriteLine("   Count    : {0}", myHT.Count)
+        Console.WriteLine($"   Count    : {0}")
```
Comment (rpetrusha) [inline snippets/visualbasic/VS_Snippets_CLR_Classic/classic Hashtable.Clear Example/VB/source.vb]:
```suggestion
        Console.WriteLine($"   Count    : {myHT.Count}")
```

**Comment by Youssef1313** [comment]
Thanks, @rpetrusha :)
I've committed it. What about the commits with message `Fixed wrong output`, should I apply same changes to the C# sample?

**Comment by rpetrusha** [comment]
Thanks for the additional changes, @Youssef1313. I'll merge your PR now.

**Pull request merged**